packages/documentation/copy/en/get-started/TS for Functional Programmers.md

```diff
@@ -586,6 +586,20 @@ a.push(102); // error
 a[0] = 101; // error
 ```
 
+It's important to remember that TypeScript doesn't transpile the immutability directives to the run-time:
```

Comment:
I don't know that we should add this; it's technically true that one can use `freeze` to achieve this behavior, but it's exceedingly rare and does have some footguns which can make it undesirable.

Additionally, inserting this section here makes the last sentence of the section a little awkward, as it's intending to refer to the previous content, not `freeze`.